cibuild.sh

```diff
@@ -61,7 +62,7 @@ run_msbuild()
     
     for i in `seq 1 $RETRY_COUNT`
     do
-        o=$(mono packages/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/MSBuild.exe /v:m /p:SignAssembly=false /p:DebugSymbols=false "$@")
+        o=$(mono $MONO_ARGS packages/Microsoft.Build.Mono.Debug.14.1.0.0-prerelease/lib/MSBuild.exe /v:m /p:SignAssembly=false /p:DebugSymbols=false "$@")
```

Comment:
That's fine.  I had hidden it when I was testing because I was losing the signal in the noise.  Can you undo this in your PR?